### ISSUE TITLE
Typography: secondary-text should be grey-5

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#1054]
Secondary-text is currently set as grey 4, it should be grey 5.

<img width="245" alt="Screen Shot 2020-03-22 at 2 10 45 PM" src="https://user-images.githubusercontent.com/38065/77261195-037b1880-6c4a-11ea-9441-7f982f686839.png">

**Comment by ianmcburnie** [comment]
Fixes in 10.2.0
